src/app/frontend/replicasetdetail/replicasetdetail.html

```diff
@@ -14,27 +14,145 @@
+        </h1>
+        <h1 class="md-caption kd-font-2" ng-hide="ctrl.replicaSetDetail.externalEndpoints">none</h1>
+      </div>
+    </div>
   </div>
-</div>
+  <md-tabs flex md-border-bottom md-dynamic-height>
+    <md-tab label="Pods">
+      <md-content class="md-padding">
+        <table class="kd-replicasetdetail-table" cellspacing="0" cellpadding="15">
+          <thead>
+          <tr>
+            <th>Pod<i class="material-icons">help</i></th>
+            <th>Start time<i class="material-icons">help</i></th>
+            <th>
+              <i class="material-icons">arrow_downward</i>
+              IP
+              <i class="material-icons">help</i>
+            </th>
+            <th>Node<i class="material-icons">help</i></th>
+          </tr>
+          </thead>
+          <tbody>
+          <tr ng-repeat="pod in ctrl.replicaSetDetail.pods">
+            <td>{{pod.name}}</td>
+            <td>{{pod.startTime | date:'short'}}</td>
+            <td>{{pod.podIP}}</td>
+            <td>{{pod.nodeName}}</td>
+          </tr>
+          </tbody>
+        </table>
+      </md-content>
+    </md-tab>
+    <md-tab label="Events">
+      <md-content flex class="md-padding">
+        <div flex layout="row">
+          <md-input-container>
+            <label>Type</label>
+            <!-- TODO(maciaszczykm): Handle value change. -->
+            <md-select ng-model="ctrl.eventType" required>
+              <md-option ng-repeat="type in ctrl.eventTypeFilter" ng-value="type">
+                {{type}}
+              </md-option>
+            </md-select>
+          </md-input-container>
+          <md-input-container>
+            <label>Source</label>
+            <!-- TODO(maciaszczykm): Handle value change. -->
+            <md-select ng-model="ctrl.eventSource" required>
+              <md-option ng-repeat="source in ctrl.eventSourceFilter" ng-value="source">
+                {{source}}
+              </md-option>
+            </md-select>
+          </md-input-container>
+        </div>
+        <table flex class="kd-replicasetdetail-table" cellspacing="0" cellpadding="15">
+          <thead>
+          <tr>
+            <th>Message
+              <i class="material-icons">help</i></th>
+            <th>Source<i class="material-icons">help</i></th>
+            <th>Sub-object<i class="material-icons">help</i></th>
+            <th>Count<i class="material-icons">help</i></th>
+            <th>First seen<i class="material-icons">help</i></th>
+            <th>
+              <i class="material-icons">arrow_downward</i>
+              Last seen
+              <i class="material-icons">help</i>
+            </th>
+          </tr>
+          </thead>
+          <tbody>
+          <tr ng-repeat="event in ctrl.replicaSetEvents.events">
+            <td class="kd-replicasetdetail-table-dark">
+              {{ event.message | limitTo: 50 }}{{event.message.length > 50 ? '...' : ''}}
```

Comment:
Yes, I will refactor it. There is still a lot to do with table styling, most of all with cells size.